src/libs/cropOrRotateImage/index.ts

```diff
@@ -40,26 +36,22 @@ function rotateCanvas(canvas, degrees) {
     const context = result.getContext('2d');
 
     // In order to rotate image along its center we have to apply next transformation
-    context.translate(result.width / 2, result.height / 2);
+    context?.translate(result.width / 2, result.height / 2);
 
     const radians = (degrees * Math.PI) / 180;
-    context.rotate(radians);
+    context?.rotate(radians);
 
-    context.drawImage(canvas, -canvas.width / 2, -canvas.height / 2, canvas.width, canvas.height);
+    context?.drawImage(canvas, -canvas.width / 2, -canvas.height / 2, canvas.width, canvas.height);
```

Comment:
Since we have 3 fn calls here, I think in this case I would prefer checking for `context`

`if (context) { ... }`

Then we don't need to use optional chaining `?`

Reply:
Not a blocker though, just personal preference. I can also see the point of leaving optional chaining to keep the pattern with the other functions in this file. You can ignore the above comment if you want 👍🏻

Reply:
@thiagobrez suggestion applied